config/routes.rb

```diff
@@ -2,7 +2,8 @@
 Rails.application.routes.draw do
   mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
   devise_for :users
-  root 'products#index'
+  root 'welcome#index'
   resources :products
   resources :users
+  resources :welcome
```

Comment:
No deberias poner resources ya que no estas agregando ninguno de los metodos.

`Resource routing allows you to quickly declare all of the common routes for a given resourceful controller. Instead of declaring separate routes for your index, show, new, edit, create, update and destroy actions, a resourceful route declares them in a single line of code.`

Reply:
listo